tex/chapters/03-Theory.tex

```diff
@@ -3,88 +3,152 @@
-Given a history for an opponent, the responses of the original strategy and the dual would be opposite.
 
-\begin{definition}\label{def:dual}
-Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
-\end{definition}
 
-It is important to note that this is different to taking a strategy and flipping it's responses.
-The dual relies on knowledge of the underlying state of the original strategy, whereas the flip does not.
-This is shown in Table \ref{tab:strat-dual-flip}.
-For an outline of how Pavlov plays, see Section \ref{ssec:strat_pavlov}.
 
 \section{The Flip}\label{sec:flip}
 The Flip is a simple transformation that returns the opposite of the strategy.
-This is subtly different from the Dual as demonstrated below.
+This is subtly different from the Dual as demonstrated in Table \ref{tab:strat-dual-flip}.
 
 \begin{definition}\label{def:flip}
 Strategy $A'$ is said to be the \textbf{Flip} of strategy $A$ if, $A$ and $A'$ return different actions for the same game history (includes opponent and self).
 \end{definition}
 
+
+
+
+\section{The Dual}\label{sec:dual}
+The Dual is another, more complex transformation of a strategy (note that both
+these transformations can be applied commutatively).
+
+\begin{definition}\label{def:dual}
+Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
+\end{definition}
+
+It is important to note that this is different to taking a strategy and flipping its responses.
+The dual relies on knowledge of the underlying state of the original strategy, whereas the flip does not.
+This is shown in Table \ref{tab:strat-dual-flip}.
+
 \begin{table}[htbp]
-\centering
-\begin{tabular}{l l l l}
-\toprule
-Opponent & Pavlov & Dual & Flip \\
-\midrule
-C & C & D & D \\
-D & C & D & C \\
-D & D & C & C \\
-C & C & D & C \\
-C & C & D & D \\
-D & C & D & C \\
-C & D & C & C \\
-D & D & C & D \\
-  & C & D & D \\
-\bottomrule
-\end{tabular}
-\caption{The different responses of Pavlov, Pavlov's Dual and Flipped Pavlov}
-\label{tab:strat-dual-flip}
+    \centering
+    \begin{tabular}{l l l l}
+        \toprule
+        Opponent history & Pavlov & Dual & Flip \\
+        \midrule
+        C & C & D & D \\
+        D & C & D & C \\
+        D & D & C & C \\
+        C & C & D & C \\
+        C & C & D & D \\
+        D & C & D & C \\
+        C & D & C & C \\
+        D & D & C & D \\
+          & C & D & D \\
+        \bottomrule
+    \end{tabular}
+    \caption{The different responses of Pavlov, Pavlov's Dual and Flipped Pavlov}
+    \label{tab:strat-dual-flip}
 \end{table}
 
 The subtle difference between Dual and Flip can be highlighted further by inspecting each row individually.
```

Comment:
not sure what this would look like, do you mean the Dual generally? or pick a strategy and draw it's dual?